plugins/callback/counter_enabled.py

```diff
@@ -21,7 +21,7 @@
     extends_documentation_fragment:
       - default_callback
     requirements:
-      - set as stdout callback in ansible.cfg  (stdout_callback = counter_enabled)
+      - set as stdout callback in ansible.cfg (C(stdout_callback = counter_enabled))
```

Comment:
Maybe 
```suggestion
      - set as stdout callback in C(ansible.cfg) (C(stdout_callback = counter_enabled))
```
On the grounds of being a filename?